vendor/vendor.json

```diff
@@ -21,7 +21,7 @@
 			"revisionTime": "2016-07-26T15:08:25Z"
 		},
 		{
-			"checksumSHA1": "dIidE/pUkqulIvTkuMBTpH3/lnU=",
+			"checksumSHA1": "A3v5qAlaOKeq3c+yDCL4jzAB0UA=",
```

Comment:
I ran `make govendor` in the assumption that it would install govendor and download the dependencies.

I guess it also updated them?